src/pages/kwakjongbum/Login/KwakLogin.js

```diff
@@ -1,7 +1,81 @@
 import React from 'react';
+import './KwakLogin.scss';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 function KwakLogin() {
-  return <div>곽종범 로그인페이지</div>;
+  const navigate = useNavigate();
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const [btn, setBtn] = useState(true);
+  const valCheck = id.includes('@') && pw.length > 4;
+
+  const inputId = e => {
+    setId(e.target.value);
+  };
+
+  const inputPw = e => {
+    setPw(e.target.value);
+  };
+
+  const btnActive = () => {
+    valCheck ? setBtn(false) : setBtn(true);
+  };
+
+  const goToMain = () => {
+    if (valCheck) {
+      navigate('/KwakMain');
+    }
+  };
+
+  return (
+    <div className="login-wrap">
+      <h1>westagram</h1>
+      <div className="form-wrap">
+        <div className="form-input">
+          <input
+            type="text"
+            name="id"
+            placeholder="전화번호, 사용자 이름 또는 이메일"
+            className="input-id"
+            onChange={inputId}
+            onKeyUp={btnActive}
+            onKeyPress={e => {
+              if (e.key === 'Enter') {
+                goToMain();
+              }
```

Comment:
```suggestion
            onKeyPress={e => {
              if (e.key === 'Enter' && btn ==='false') {
                goToMain();
              }
```
이렇게 수정하면 유효성검사를 통과 했을때만 메인페이지로 이동할 수 있을거 같습니다.